tests/v2/validation/deleting/delete_init_machine_test.go

```diff
@@ -0,0 +1,44 @@
+package deleting
+
+import (
+	"testing"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type DeleteInitMachineTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+}
+
+func (d *DeleteInitMachineTestSuite) TearDownSuite() {
+	d.session.Cleanup()
+}
+
+func (d *DeleteInitMachineTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	d.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(d.T(), err)
+
+	d.client = client
+}
+
+func (d *DeleteInitMachineTestSuite) TestDeleteInitMachine() {
+	clusterID, err := clusters.GetClusterIDByName(d.client, d.client.RancherConfig.ClusterName)
+	require.NoError(d.T(), err)
+
+	deleteInitMachine(d.T(), d.client, clusterID)
+} 
```

Comment:
since this says `machine` I assume that's just for k3s/rke2. But can we make that more explicit? Following some of the new code added to have better naming in Qase, something like this 
```
		clusterID, err := clusters.GetV1ProvisioningClusterByName(s.client, s.client.RancherConfig.ClusterName)
		require.NoError(s.T(), err)

		cluster, err := tt.client.Steve.SteveType(ProvisioningSteveResourceType).ByID(clusterID)
		require.NoError(s.T(), err)

		updatedCluster := new(apisV1.Cluster)
		err = v1.ConvertToK8sType(cluster, &updatedCluster)
		require.NoError(s.T(), err)

		if strings.Contains(updatedCluster.Spec.KubernetesVersion, "rke2") {
			tt.name = "Replacing RKE2 " + tt.name
		} else {
			tt.name = "Replacing K3S " + tt.name
		}

		s.Run(tt.name, func() {
			deleteInitMachine(d.T(), d.client, clusterID)
	}
```

Reply:
this is just for rke2/k3s 🙂

Reply:
still, I think we need to include the cluster type in the test name. Can you update the test function to do that, similar to the snippet above? this helps with Qase reporting.